### PR TITLE
(DOCSP-16884) Backfills docs content for invitations and users commands

### PR DESCRIPTION
**Pull request by sarahsimpers**
## Proposed changes

Backfills docs content for invitations commands and users commands

**I made one change where we were showing org roles and I think we should be showing project roles, please let me know if I was incorrect in my assessment on that**

_Jira ticket:_

https://jira.mongodb.org/browse/DOCSP-17364
https://jira.mongodb.org/browse/DOCSP-17371
https://jira.mongodb.org/browse/DOCSP-17345
https://jira.mongodb.org/browse/DOCSP-17348
https://jira.mongodb.org/browse/DOCSP-16884
https://jira.mongodb.org/browse/DOCSP-16885
https://jira.mongodb.org/browse/DOCSP-16883
https://jira.mongodb.org/browse/DOCSP-27841

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/usage/usage.go [hunk 1]
```diff
@@ -106,12 +106,16 @@ dbName and collection are required only for built-in roles.`
...
-	FirstName                                 = "User's first name."
-	LastName                                  = "User's last name."
-	OrgRole                                   = "User's roles  for the associated organization. To learn which values the Atlas CLI accepts, see the Items Enum for roles in the API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createOrganizationInvitation/."
-	ProjectRole                               = "User's roles  for the associated project. To learn which values the Atlas CLI accepts, see the Items Enum for roles in the API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createProjectInvitation/."
+	FirstName                                 = "First or given name for the user."
+	LastName                                  = "Last name, family name, or surname for the user."
+	OrgRole                                   = "User's roles  for the associated organization. Valid values include ORG_OWNER, ORG_MEMBER, ORG_GROUP_CREATOR, ORG_BILLING_ADMIN, and ORG_READ_ONLY."
```
Comment (lmkerbey-mdb) [inline internal/usage/usage.go:111]: **TYPO**: There's an extra space after 'roles'.

```suggestion
	OrgRole                                   = "User's roles for the associated organization. Valid values include ORG_OWNER, ORG_MEMBER, ORG_GROUP_CREATOR, ORG_BILLING_ADMIN, and ORG_READ_ONLY."
```

File: internal/usage/usage.go [hunk 2]
```diff
@@ -106,12 +106,16 @@ dbName and collection are required only for built-in roles.`
...
+	UserOrgRole                               = "Unique 24-digit string that identifies the organization, colon, and the user's role  for the organization. Specify this value as orgID:ROLE. Valid values for ROLE include ORG_OWNER, ORG_MEMBER, ORG_GROUP_CREATOR, ORG_BILLING_ADMIN, and ORG_READ_ONLY."
+	ProjectRole                               = "User's roles  for the associated project. Valid values include GROUP_CLUSTER_MANAGER, GROUP_DATA_ACCESS_ADMIN, GROUP_DATA_ACCESS_READ_ONLY, GROUP_DATA_ACCESS_READ_WRITE, GROUP_OWNER, and GROUP_READ_ONLY."
```
Comment (lmkerbey-mdb) [inline internal/usage/usage.go:113]: **TYPO**: Remove extra space.

```suggestion
	ProjectRole                               = "User's roles for the associated project. Valid values include GROUP_CLUSTER_MANAGER, GROUP_DATA_ACCESS_ADMIN, GROUP_DATA_ACCESS_READ_ONLY, GROUP_DATA_ACCESS_READ_WRITE, GROUP_OWNER, and GROUP_READ_ONLY."
```

File: internal/usage/usage.go [hunk 3]
```diff
@@ -106,12 +106,16 @@ dbName and collection are required only for built-in roles.`
...
 	MCLIOrgRole                               = "User's roles  for the associated organization. To learn which values MCLI accepts, see the Items Enum for roles in the Atlas API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createOrganizationInvitation/."
-	MCLIProjectRole                           = "User's roles  for the associated project. To learn which values MCLI accepts, see the Items Enum for roles in the Atlas API spec: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createProjectInvitation/. Additionally, MCLI accepts the following values: GROUP_AUTOMATION_ADMIN, GROUP_BACKUP_ADMIN, and GROUP_MONITORING_ADMIN."
+	MCLIProjectRole                           = "User's roles  for the associated project. Valid values include GROUP_CLUSTER_MANAGER, GROUP_DATA_ACCESS_ADMIN, GROUP_DATA_ACCESS_READ_ONLY, GROUP_DATA_ACCESS_READ_WRITE, GROUP_OWNER, GROUP_READ_ONLY, GROUP_AUTOMATION_ADMIN, GROUP_BACKUP_ADMIN, and GROUP_MONITORING_ADMIN."
```
Comment (lmkerbey-mdb) [inline internal/usage/usage.go:117]: **TYPO**: Remove extra space.

```suggestion
	MCLIProjectRole                           = "User's roles for the associated project. Valid values include GROUP_CLUSTER_MANAGER, GROUP_DATA_ACCESS_ADMIN, GROUP_DATA_ACCESS_READ_ONLY, GROUP_DATA_ACCESS_READ_WRITE, GROUP_OWNER, GROUP_READ_ONLY, GROUP_AUTOMATION_ADMIN, GROUP_BACKUP_ADMIN, and GROUP_MONITORING_ADMIN."
```

**Review by lmkerbey-mdb — COMMENTED** [review]
Three small typos, then LGTM.

File: internal/cli/iam/organizations/invitations/invite.go [hunk 4]
```diff
@@ -70,12 +71,14 @@ func InviteBuilder() *cobra.Command {
 	opts.Template = createTemplate
 	cmd := &cobra.Command{
 		Use:     "invite <email>",
-		Short:   "Invites one user to the organization that you specify.",
+		Short:   "Invite the specified MongoDB user to your organization.",
```
Comment (gssbzn) [inline internal/cli/iam/organizations/invitations/invite.go:74]: just in case this is cloud user and not db user, mongodb user let's this a bit ambiguous

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**